source/1_engine/rex_engine/src/diagnostics/logging/logger_config.cpp

```diff
@@ -114,11 +121,24 @@ namespace rexlog
           }
         }
 
-        details::Registry::instance().set_levels(rsl::move(levels), global_level_found ? &global_level : nullptr);
+        rexlog::details::Registry::instance().set_levels(rsl::move(levels), global_level_found ? &global_level : nullptr);
       }
 
     } // namespace helpers
-  }   // namespace cfg
-} // namespace rexlog
 
-// NOLINTEND(misc-definitions-in-headers)
+    // search for REXLOG_LEVEL= in the args and use it to init the levels
+    void load_log_levels(const rsl::vector<rsl::string_view>& args)
+    {
+      const rex::DebugString rexlog_level_prefix("REXLOG_LEVEL=");
+
+      for(rsl::string_view s : args)
+      {
+        if(s.find(rexlog_level_prefix) == 0)
+        {
+          auto levels_string = s.substr(rexlog_level_prefix.size());
+          helpers::load_levels(rex::DebugString(levels_string));
```

Comment:
the interface of load_levels, doesn't have to take a rsl::DebugString, a string_view would work just fine, and then you can avoid a possible allocation here

Reply:
How am I supposed to create a DebugStringStream from a rsl::string_view?